scripts/ralph-loop.sh

```
@@ -137,17 +137,17 @@ show_status() {
   fi
   echo ""
 
-  # PROGRESS.md status
-  if [[ -f "$PROJECT_ROOT/PROGRESS.md" ]]; then
+  # docs/_ralphwiggum/PROGRESS.md status
+  if [[ -f "$PROJECT_ROOT/docs/_ralphwiggum/PROGRESS.md" ]]; then
     local total
     local done
```

Shellcheck (0.11.0)

[warning] Use semicolon or linefeed before 'done' (or quote to make it literal).

(SC1010)